packages/iotex-react-block-producers/src/bp-render.tsx

```diff
@@ -26,9 +26,7 @@ export function renderDelegateName(
     <a href={`/delegate/${record.id}`}>
       <div style={{ display: "flex", alignItems: "center" }}>
         <div>
-          {record.serverStatus !== DServerStatus.NOT_EQUIPPED && (
```

Comment:
why not show `NOT_EQUIPPED ` status before?

Reply:
we don't display dot if it's not equipped

Reply:
```
if slashed:
  display yellow dot
else if not equipped
  display nothing
else
   display colored
```